### PR TITLE
doc: 16.2.11 Release Notes

### PR DESCRIPTION
**Pull request by yuriw**
See https://tracker.ceph.com/issues/58257#note-2 for details

Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by anthonyeleven — APPROVED** [review]

**Comment by anthonyeleven** [comment]
Should be `doc/releases:` I think

**Comment by anthonyeleven** [comment]
The whitespace changes look legit to me, ymmv

**Review by jdurgin — APPROVED** [review]

**Comment by idryomov** [comment]
> The whitespace changes look legit to me, ymmv

It's legit, otherwise I wouldn't have approved ;)  It just shouldn't be bundled with other changes.

File: doc/releases/pacific.rst [hunk 1]
```diff
@@ -2,6 +2,529 @@
 Pacific
 =======
 
+v16.2.11 Pacific
+================
+
+This is the eleventh backport release in the Pacific series.
+
+Notable Changes
+---------------
```
Comment (neha-ojha) [inline doc/releases/pacific.rst:11]: nit: it will be nice to add a blank line here and between the bulleted items

**Review by neha-ojha — APPROVED** [review]

**Review by nizamial09 — APPROVED** [review]

File: doc/releases/pacific.rst (hunk 1 above)
Comment (ljflores) [reply doc/releases/pacific.rst:11]: @yuriw this still needs to be fixed ^

**Review by ljflores — APPROVED** [review]

**Pull request merged**